### PR TITLE
vue electron demo

### PR DESCRIPTION
**Pull request by ksyeo1010**
Please check medium article: https://medium.com/picovoice/its-time-for-local-speech-recognition-df7c911fe944 as you go over this as well.

### CODE REVIEW

File: demo/electron/src/components/VoiceTimer.vue [hunk 1]
```diff
@@ -1,143 +1,221 @@
...
-    pause: function () {
-      if (this.$refs.picovoice.pause()) {
-        this.isListening = !this.isListening;
+    };
+
+    watch(
+      () => state.wakeWordDetection,
+      (wakeWordDetection) => {
+        if (wakeWordDetection !== null) {
+          engine.value = "rhn";
```
Comment (ErisMik) [inline demo/electron/src/components/VoiceTimer.vue:117]: I don't know much about ow the Picovoice Vue SDK works but I feel like this should be handled within the SDK no?

**Review by ErisMik — APPROVED** [review]
I guess we will also need to update the github commit links in the article

File: demo/electron/src/components/VoiceTimer.vue (hunk 1 above)
Comment (ksyeo1010) [reply demo/electron/src/components/VoiceTimer.vue:117]: I don't think we handle it inside sdks

Comment (ksyeo1010) [reply demo/electron/src/components/VoiceTimer.vue:117]: nvm I think I know wym, can handle it without setting the value

**Comment by ksyeo1010** [comment]
> I guess we will also need to update the github commit links in the article

I think I might just remove the links, doesnt make sense now

**Comment by ErisMik** [comment]
I'll merge after links are fixed

**Pull request merged**